kratos/sources/variables.cpp

```diff
@@ -586,7 +588,8 @@ namespace Kratos
 
   void KratosApplication::RegisterVariables()
   {
-
+        RegisterLinearSolvers();
```

Comment:
i am not sure if this is the correct place, however we need somewher that is surely called at initialization time